include/robot_localization/navsat_transform.hpp

```diff
@@ -166,11 +166,31 @@ class NavSatTransform
    */
   bool broadcast_utm_transform_as_parent_frame_;
 
+  /**
+   * @brief TimerBase for publish callback
+   */
+  rclcpp::Service<robot_localization::srv::SetDatum>::SharedPtr datum_srv;
```

Comment:
need the trailing `_` for class variables by the projects style. Same with `filtered_gps_pub`, `gps_odom_pub` `gps_sub` `imu_sub` `odom_sub`